src/main/java/org/tarantool/TarantoolClientImpl.java

```diff
@@ -337,25 +356,21 @@ private boolean directWrite(ByteBuffer buffer) throws InterruptedException, IOEx
     }
 
     protected void readThread() {
-        try {
-            while (!Thread.currentThread().isInterrupted()) {
-                try {
-                    TarantoolPacket packet = ProtoUtils.readPacket(readChannel);
+        while (!Thread.currentThread().isInterrupted()) {
+            try {
```

Comment:
No behaviour changed here, yep? You just make readThread() looks more similar to writeThread()?

Reply:
correct